src/js/pages/jobs/JobTaskDetailPage.js

```diff
@@ -20,19 +26,25 @@ class JobTaskDetailPage extends React.Component {
       { label: "Logs", routePath: routePrefix + "/logs" }
     ];
 
+    const job = MetronomeStore.getJob(id);
     const task = MesosStateStore.getTaskFromTaskID(taskID);
 
-    let breadcrumbs;
+    let breadcrumbs = (
+      <JobsBreadcrumbs tree={DCOSStore.jobTree} item={job} details={false} />
```

Comment:
Do you think it makes sense to have 2 components and pick one when we need details and another when we don't? Right now `details=false` doesn't tell me anything. Let's think how we can remove this flag.

ℹ️ https://martinfowler.com/bliki/FlagArgument.html